setup.cfg

```
@@ -0,0 +1,21 @@
+[metadata]
+name = DB12
+description = DIRAC Benchmark 2012
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/pypa/sampleproject
```

Oops, you forgot to replace the example name by the project name I guess :-)

```suggestion
url = https://github.com/DIRACGrid/DB12
```